orga/tests/helpers/extended-ember-test-helpers/click-by-label.js

```diff
@@ -0,0 +1,35 @@
+import { click, findAll } from '@ember/test-helpers';
+
+export default function clickByLabel(labelText) {
+  const clickableElement = _findClickableElementForLabel(labelText);
+
+  if (!clickableElement) {
+    throw new Error(`Cannot find clickable element labelled "${labelText}".`);
+  }
+
+  return click(clickableElement);
```

Comment:
**suggestion:** J'aime bien éviter les négations, ça m'évite ce genre de confusions quand ça devient compliqué 
![](https://media.giphy.com/media/zkSFsZpQMZuG4/giphy.gif)

Quelque chose comme ça du coup
```js 
  if (clickableElement) {
	return click(clickableElement);
  }

  throw new Error(`Cannot find clickable element labelled "${labelText}".`);
```

Reply:
Ok that i can do